src/Pages/NotificationsSettings/KeywordsSettingForm.tsx

```diff
@@ -0,0 +1,141 @@
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const [selectedStacks, setSelectedStacks] = useState<Stack[]>(getSelectedStacks(values));
+
+  const { setValue, watch, handleSubmit } = useForm<RecruitmentKeywordsForm>({
+    defaultValues: {
+      stackIds: [],
+      positionIds: [],
+      categoryIds: [],
+    },
+    values,
+  });
+
+  const handleSelectedStacks = (stacks: Stack[]) => {
+    setSelectedStacks(stacks);
+    setValue('stackIds', [...stacks.map((stack: Stack) => stack.id)]);
+  };
+
+  return (
+    <Form
+      onSubmit={handleSubmit(({ stackIds, positionIds, categoryIds }) =>
+        editKeywordsMutate({ stackIds, positionIds, categoryIds }),
+      )}
+    >
+      <KeywordsSettingBox>
+        <KeywordForm label="기술 스택" type="stackIds">
+          <ChipMenu
+            key={'기술 스택'}
+            checked={watch('stackIds').length !== 0}
+            onClick={() => setIsOpen((prev) => !prev)}
+          >
+            기술 스택
+          </ChipMenu>
+          {isOpen && (
+            <StackModal
+              handleModal={setIsOpen}
+              initialSelectedStacks={selectedStacks}
+              handleSelectedStacks={handleSelectedStacks}
+            />
+          )}
+        </KeywordForm>
+        <KeywordForm label="포지션" type="positionIds">
+          {POSITION_KEYWORD_OPTIONS.map(({ value: id, label }) => (
+            <ChipMenu
+              key={label}
+              checked={watch('positionIds').includes(id)}
+              onClick={() => setValue('positionIds', handleValues(watch('positionIds'), id))}
+              id={id}
+            >
+              {label}
+            </ChipMenu>
+          ))}
+        </KeywordForm>
+        <KeywordForm label="카테고리" type="categoryIds">
+          {CATEGORY_KEYWORD_OPTIONS.map(({ value: id, label }) => (
+            <ChipMenu
+              key={label}
+              checked={watch('categoryIds').includes(id)}
+              onClick={() => setValue('categoryIds', handleValues(watch('categoryIds'), id))}
+              id={id}
+            >
+              {label}
+            </ChipMenu>
+          ))}
+        </KeywordForm>
+      </KeywordsSettingBox>
+      <BtnsBox>
+        <Button
+          type="button"
+          size="fullWidth"
+          onClick={() => {
+            setValue('stackIds', [...values.stackIds]);
+            setValue('positionIds', [...values.positionIds]);
+            setValue('categoryIds', [...values.categoryIds]);
+          }}
+          disabled={disabled}
+        >
+          취소
+        </Button>
+        <Button size="fullWidth" scheme="secondary" disabled={disabled}>
+          저장
+        </Button>
+      </BtnsBox>
+    </Form>
+  );
+};
+
+const KeywordsSettingBox = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 16px;
+  align-self: stretch;
+`;
+
+const Form = styled.form`
+  display: flex;
+  padding: 0 0 32px 0;
```

Comment:
```suggestion
  padding-bottom: 32px;
```
padding이 아래에만 들어갈 경우 padding-bottom을 사용하는게 더 적절할 것 같습니다!

Reply:
수정하였습니다.